Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'rails', '~> 5.2.0.rc1'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'coveralls', require: false
+gem 'devise'
+gem 'turbolinks'
```

Comment:
Please make sure you add the 'puma' gem back to your Gemfile

Reply:
We have that gem in develop branch, just merged it